data/en/filemove.json

```diff
@@ -18,9 +18,9 @@
 	"links": [], 
     	"examples": [
 		{
-			"title": "Script Syntax",
+			"title": "Move file from here to there",
 			"description": "",
-			"code": "<cfscript>\r\nfileMove(#sourcefile#, #destinationfile#);\r\n</cfscript>",
+			"code": "fileMove(#sourcefile#, #destinationfile#);",
```

Comment:
Thanks for removing cfscript - The pound signs can be removed here too, so just `fileMove(sourcePath, destinationPath)`

Reply:
Sure. Looks like I have been a bit inattentive^^